README.md

```diff
@@ -48,16 +48,13 @@ metadata:
     replicated.com/nfs: ObjectStore
     replicated.com/hostpath: ObjectStore
 data:
-  # Customize these values
-  veleroDeploymentName: velero
-  resticDaemonsetName: restic
```

Comment:
velero does not allow configuring the deployment/daemonset names, and this makes it harder to support multiple velero versions, so i removed them

Reply:
makes sense if velero doesn't support that